### PR TITLE
:memo: Update descriptions

### PR DESCRIPTION
**Pull request by cowleyk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 HTTPRequests
 ============
-A Block that makes HTTP requests.  One request is made for every input signal. For each successful request, an output signal is created with the url that was hit.
+The HTTPReqeusts block sends an HTTP request for each item in the list of incoming signals. For each successful request, an output signal is created that includes the response from the request.
```
Comment (jennyknuth) [inline README.md]: Fix typo in block name. 

Is "item" ambiguous here? What is the best way to describe signals, lists of signals, and streams of signals? I think this change is good, but as we go through, let's try to come up with a clear and consistent way to refer to these things.

File: README.md [hunk 2]
```diff
@@ -68,12 +69,12 @@ NOTE: This example is superseded by the [NioCommand](https://github.com/nio-bloc
 
 HTTPRequestsPostSignal
 ======================
-Similar to the HTTPRequests block.  One request is made for every signal input.  The input signal will be used as the body of the post request.
+The HTTPRequestsPostSignal block is similar to the HTTPRequests block.  One request is made for every signal input.  The input signal will be used as the body of the post request.
 
 Properties
 ----------
 - **basic_auth_creds**: When making a request that needs Basic Authentication, enter the username and password.
```
Comment (jennyknuth) [inline README.md]: Optionally enter the username and password when making a request that requires basic authentication.

File: README.md [hunk 3]
```diff
@@ -68,12 +69,12 @@ NOTE: This example is superseded by the [NioCommand](https://github.com/nio-bloc
...
-- **enrich**: If true, the output signal will include the original signal sent into the block
+- **enrich**: If false (unchecked), the output signal will include the original signal sent into the block
```
Comment (jennyknuth) [inline README.md]: Another idea: 
If checked (true), the original signal sent into the block will be excluded from the signal sent out of the block. If unchecked (false), the output signal will include the original signal sent into the block.

File: README.md [hunk 4]
```diff
@@ -68,12 +69,12 @@ NOTE: This example is superseded by the [NioCommand](https://github.com/nio-bloc
 
 HTTPRequestsPostSignal
 ======================
-Similar to the HTTPRequests block.  One request is made for every signal input.  The input signal will be used as the body of the post request.
+The HTTPRequestsPostSignal block is similar to the HTTPRequests block.  One request is made for every signal input.  The input signal will be used as the body of the post request.
```
Comment (jennyknuth) [inline README.md]: Add link to HTTPRequests block?

File: README.md (hunk 3 above)
Comment (jennyknuth) [reply README.md]: What reads better? 
signal sent into the block | input signal | input sent into the block | input signal sent into the block
signal sent out from the block | output signal | output sent out by the block | output signal sent out by the block

Comment (cowleyk) [reply README.md]: I like 'input signal' and 'output signal'.  I feel that should be sufficient for someone reading the docs.

Comment (cowleyk) [reply README.md]: However in this context, I think the more explicit 'original signal sent into the block' is needed to explain the property

File: README.md [hunk 5]
```diff
@@ -1,27 +1,27 @@
...
-- **headers**: Custom Headers. Headers and Values can be simple strings or Expression Properties to use incoming signals.
-- **http_method**: HTTP Method (e.g. GET|POST|PUT|DELETE).
-- **require_json**: If True and response is not json, log warning and do not emit a signals. If False and response is not json, emit a signal of format {'raw': response.text}.
-- **retry_options**: Configurables for retrying to connect on request failure.
-- **timeout**: Timeout in seconds for each request, if empty or 0 then requests will not time out.
-- **url**: URL to make request to.
-- **verify**: For https, check a host's SSL certificate. Default value for the block is True, the same as the requests library.
+- **basic_auth_creds**: When making a request that needs Basic Authentication, enter the username and password.
+- **data**: URL parameters are key-value pairs that can appear in a URL path. Keys and values can be either simple strings or expression properties that use incoming signals.
```
Comment (cowleyk) [inline README.md]: I like this a lot!

**Pull request merged**